souto-app/src/components/pages/Pickit.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from "react";
+import {
+  Accordion,
+  AccordionSummary,
+  AccordionDetails,
+  Typography,
+  Avatar,
+} from "@material-ui/core";
+import { makeStyles, withStyles } from "@material-ui/core/styles";
+
+const style = {
+  pickBox: {
+    marginBottom: "8px",
+    marginLeft: "10px",
+    position: "relative",
+    top: "20px",
+    width: "90%",
+    borderRadius: "15px",
+  },
+  title: {
+    transform: "translateY(17%)",
+    fontSize: "small",
+    color: "grey",
+    marginLeft: "10px",
+  },
+  avatar: {
+    width: "25px",
+    height: "25px",
+    backgroundColor: "purple",
+  },
+};
+
+export default class Pickit extends Component {
```

Comment:
PIckit its not a good name.
Pick its the action
ticket its the subject

Change it to ticket